sys/log/src/log_nmgr.c

```diff
@@ -180,7 +431,7 @@ log_nmgr_clear(struct nmgr_jbuf *njb)
         }
 
         rc = log_flush(log);
-        if (rc != 0) {
+        if (rc) {
```

Comment:
Why the change here -- same use: the majority of the code is written the way it was previously checked.